serverless.yml

```diff
@@ -281,6 +353,14 @@ plugins:
 
 resources:
   Resources:
+    MyTopic:
+      Type: AWS::SNS::Topic
+      Properties:
+        DisplayName: ${self:custom.cfg.vautomatorSNS}
+        TopicName: ${self:service}-${self:provider.stage}-vautomator-results-topic
+        Subscription:
+          - Protocol: email
+            Endpoint: culucenk@mozilla.com
```

Comment:
Can we add multiple emails here? Like say you, kang, and myself?

Reply:
Based on my research, yes. I will try to deploy with multiple emails.